proxies/live/apiproxy/policies/GenerateJWT.NrlfClientAssertion.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<GenerateJWT async="false" continueOnError="false" enabled="true" name="GenerateJWT.NrlfClientAssertion">
+  <DisplayName>GenerateJWT.NrlfClientAssertion</DisplayName>
+  <Algorithm>RS512</Algorithm>
+  <PrivateKey>
+    <Id>test-1</Id>
+    <Value ref="private.nrlfPrivateKey"/>
+  </PrivateKey>
+  <Subject ref="nrlfClientId"/>
+  <Issuer ref="nrlfClientId"/>
+  <Audience>https://int.api.service.nhs.uk/oauth2/token</Audience>
```

Comment:
Can we not add this to the `booking-and-referrals-config` KVM (with the Client ID) rather than hardcoding it? Will help us when we release to different ENVs